controllers/volumereplicationgroup_controller.go

```diff
@@ -679,6 +746,13 @@ func (v *VRGInstance) reconcileVRForDeletion(pvc *corev1.PersistentVolumeClaim,
 		return requeue
 	}
 
+	if err := v.deletePVFromS3Stores(pvc, log); err != nil {
```

Comment:
(initial comment) We should delete PVs from the S3 store only when being deleted as primary.

As Secondary we may only have  a single S3 store, and we should not delete the PVs from there, as it is a single copy.

The downside is that we would need to attempt to delete PVs in all stores, local and remote (from a restore perspective) but that is possibly fine as we backup everywhere as well.

Reply:
Right.  Thanks